package.json

```diff
@@ -10,7 +10,13 @@
     "index.*"
   ],
   "peerDependencies": {
-    "eslint": ">=5.0.0"
+    "eslint": ">=5.0.0",
+    "pug-lexer": ">=5.0.0"
+  },
+  "peerDependenciesMeta": {
+    "pug-lexer": {
+      "optional": true
```

Comment:
Need to try later if it's possible this way to make the pug-lexer optional
So it's only needed when there is a `<template lang="pug">` in the project

It's also just a peer dependency cause the plugin that uses this parser will the define the "real" dependency